charts/ceph-csi-rbd/README.md

```diff
@@ -121,19 +121,23 @@ charts and their default values.
+| `provisioner.provisioner.image.extraArgs`     | Specifies extra arguments for the provisioner sidecar                                                                                                                                | `[]`                                     |
 | `provisioner.attacher.image.repository`        | Specifies the csi-attacher image repository URL                                                                                                      | `registry.k8s.io/sig-storage/csi-attacher`              |
 | `provisioner.attacher.image.tag`               | Specifies image tag                                                                                                                                  | `v4.0.0`                                           |
 | `provisioner.attacher.image.pullPolicy`        | Specifies pull policy                                                                                                                                | `IfNotPresent`                                     |
+| `provisioner.attacher.image.extraArgs`     | Specifies extra arguments for the attacher sidecar                                                                                                                                | `[]`                                     |
 | `provisioner.attacher.name`                    | Specifies the name of csi-attacher sidecar                                                                                                           | `attacher`                                         |
 | `provisioner.attacher.enabled`                 | Specifies whether attacher sidecar is enabled                                                                                                        | `true`                                             |
 | `provisioner.resizer.image.repository`         | Specifies the csi-resizer image repository URL                                                                                                       | `registry.k8s.io/sig-storage/csi-resizer`               |
 | `provisioner.resizer.image.tag`                | Specifies image tag                                                                                                                                  | `v1.6.0`                                           |
 | `provisioner.resizer.image.pullPolicy`         | Specifies pull policy                                                                                                                                | `IfNotPresent`                                     |
+| `provisioner.resizer.image.extraArgs`     | Specifies extra arguments for the resizer sidecar                                                                                                                                | `[]`                                     |
```

Comment:
just wondering the indendation part here.. looks like the allignment has gone wrong a bit .. @Sea-you

Reply:
@humblec hm, I checked both READMEs and they render fine for me, but fixed nevertheless.